boards/common/msba2/board_common_init.c

```diff
@@ -57,16 +57,16 @@ void init_clks2(void)
 
 void watchdog_init(void)
 {
-    WDCLKSEL = 0;                                   // clock source: RC oscillator
-    WDMOD &= ~WDTOF;                                // clear time-out flag
+    WDCLKSEL = 0;                                   /* clock source: RC oscillator */
```

Comment:
This one is pushed beyond 80 chars, maybe make the indentation match with line 69 (together with the comment on line 61.